config/webpack.common.js

```diff
@@ -33,7 +33,7 @@ module.exports = {
   devtool: "source-map",
   output: {
     path: path.resolve(__dirname, '../build/public'),
-    filename: "main.js",
+    filename: "[name].js",
```

Comment:
Is `[name]` resolved to key name in `entry` map?

Reply:
yep, that's right